example_scenes/basic.py

```diff
@@ -15,47 +15,47 @@
 # for a 1920x1080 video)
 
 
-class OpeningManimExample(Scene):
-    def construct(self):
-        title = TextMobject("This is some \\LaTeX")
+class OpeningManimExample(Scene): # Define a new scene named OpeningManimScene inheriting from Scene
+    def construct(self): # Define the animations to be constructed
+        title = TextMobject("This is some \\LaTeX") # Define a TextMobject, doesn't use LaTeX
```

Comment:
According to the code I don't think it's correct since TextMobject inherits from TexObject. To not use Latex, you have to use `Text` object.